pkg/catalog/ingress.go

```diff
@@ -50,3 +53,57 @@ func (mc *MeshCatalog) GetIngressRoutesPerHost(service service.MeshService) (map
 
 	return domainRoutesMap, nil
 }
+
+// GetIngressPoliciesForService returns a list of inbound traffic policies for a service as defined in observed ingress k8s resources.
+func (mc *MeshCatalog) GetIngressPoliciesForService(svc service.MeshService, sa service.K8sServiceAccount) ([]*trafficpolicy.InboundTrafficPolicy, error) {
+	inboundIngressPolicies := []*trafficpolicy.InboundTrafficPolicy{}
+	ingresses, err := mc.ingressMonitor.GetIngressResources(svc)
+	if err != nil {
+		log.Error().Err(err).Msgf("Failed to get ingress resources for service %s", svc)
+		return inboundIngressPolicies, err
+	}
+	if len(ingresses) == 0 {
+		log.Trace().Msgf("No ingress resources found for service %s", svc)
+		return inboundIngressPolicies, err
+	}
+
+	defaultWildcardRoute := trafficpolicy.HTTPRouteMatch{
+		PathRegex: constants.RegexMatchAll,
+		Methods:   []string{constants.RegexMatchAll},
+	}
+
+	ingressWeightedCluster := service.WeightedCluster{
```

Comment:
might make sense to use `getDefaultWeightedClusterForService` func here https://github.com/openservicemesh/osm/blob/6b699b938f313c9807184fa44e2f5cbcf260787a/pkg/catalog/routes.go#L401